alma/alma-installments-prestashop
# [insurance] Register hooks on update module

### Reason for change

[Linear task](https://linear.app/almapay/issue/ECOM-1499/%5B%F0%9F%A7%A9-ps%5D-handle-migration-update-with-insurance-if-we-update-the-module)

### How to test

- mount an infra without alma
- install alma from the marketplace and configure it
- build the plugin version 4.0.0
- upload the plugin in the BO
- activate insurance
- ensure all insurance flow works
